apps/hash-ai-worker-ts/src/activities/infer-entities.ts

```diff
@@ -89,6 +89,9 @@ export const inferEntities = async ({
           inheritsFrom: { outgoing: 255 },
         },
         temporalAxes: currentTimeInstantTemporalAxes,
+        // @todo Create new inferred entities as drafts instead of actual entities
+        //   see https://linear.app/hash/issue/H-1374
+        includeDrafts: false,
```

Comment:
This query doesn't request any entities (it's for types) so I don't think we need this comment
```suggestion
        includeDrafts: false,
```